testing/TestHarness/TestHarness.Shared/Ext/Core/Storage/StorageOneViewModel.cs

```diff
@@ -105,8 +106,8 @@ public async Task PerfTest()
 			var keyValues = new List<(string Key, string Value)>();
 			for (int i = 0; i < keyCount; i++)
 			{
-				var key = GenerateString(Random.Shared.Next(0, 100));
-				var value = GenerateString(Random.Shared.Next(0, 1000));
+				var key = GenerateString(_random.Next(0, 100));
+				var value = GenerateString(_random.Next(0, 1000));
```

Comment:
I'm curious, why did you change this?

Reply:
Random.Shared isn't supported on Xamarin targets